### PR TITLE
Fixes: #603 - Improve team page and add more data

### PR DESCRIPTION
**Pull request by isuruAb**
Fixes issue #603 

Changes:

- [x] Add Twitter, LinkedIn, Website or Blog to each Founder
- [x] Add section for managers
- [x] Add Twitter, LinkedIn, Website or Blog to each Manager
- [x] Sort developers by name
- [x] Add main contributors who have worked on the project before into developers section, e.g Harshit, Sudheesh, Damini, Jigyasa, Niranjan etc.
- [x] Change subinfo line to add the info about the core item, e.g. "Android Developer" instead of "Github" link
- [x] Add Twitter, LinkedIn, Website or Blog to each Developer
- [x]  Add data

Demo Link: http://isuruab.surge.sh/team

Screenshots for the change: 
screen shot 2017-07-28 at 11 17 08 am

@madhavrathi @uday96 @rishiraj824  Please review this PR

### CODE REVIEW

**Comment by uday96** [comment]
@isuruAb The demo link is not working for team page. Please check

**Comment by isuruAb** [comment]
@uday96  please check the link again. http://isuruab1.surge.sh/team

File: src/components/Team/TeamList.js [hunk 1]
```diff
@@ -2,88 +2,207 @@ var team = [{
...
-  }]
-},{
-  'bots' :[{
-    'name' : 'Manveer Singh',
+  'server': [{
+    'name': 'Saurabh Jain',
+    'github': 'https://github.com/saurabhjn76',
+    'avatar': 'https://scontent.fmaa2-1.fna.fbcdn.net/v/t1.0-1/c80.0.480.480/284797_228629740506894_1570458_n.jpg?oh=9790e3be67ff793ab29a474b915b3d15&oe=5A0AED30',
+    'designation': 'Java + Web Developer',
+    'twitter': 'https://twitter.com/mariobehling',
```
Comment (rishiraj824) [inline src/components/Team/TeamList.js]: Please check the twitter links its `/mariobehling`

File: src/components/Team/TeamList.js [hunk 2]
```diff
@@ -2,88 +2,207 @@ var team = [{
...
+    'avatar': 'https://avatars3.githubusercontent.com/u/14003244?v=4&s=460',
+    'designation': 'Android Developer',
+    'twitter': 'https://twitter.com/mariobehling',
+    'linkedin': 'https://www.linkedin.com/in/amit-tiwary-8bb081103',
+    'blog': '#'
+  },
+  {
+    'name': 'Rishi Raj',
+    'github': 'http://github.com/rishiraj824',
+    'avatar': 'https://avatars3.githubusercontent.com/u/11540785?v=4&s=460',
```
Comment (rishiraj824) [inline src/components/Team/TeamList.js]: Please do not change the image link `https://image.ibb.co/jypfZk/afa59275_0e6e_469b_9bb6_9c2050ddbf69.jpg`

Comment (mariobehling) [inline src/components/Team/TeamList.js]: Images need to be on the repository itself. We should not depend on external services and website.

Comment (rishiraj824) [inline src/components/Team/TeamList.js]: Okay, so we need to have a folder like `images/team/` containing all the images of the team members. The avatar can now have links to the images in the local repository.

**Comment by isuruAb** [comment]
@mariobehling  please review this.i have done those changes.
http://isuruab.surge.sh/team

**Comment by mariobehling** [comment]
Thank you!

**Pull request merged**